plugin-jdbc-rockset/src/main/java/io/kestra/plugin/jdbc/rockset/Trigger.java

```diff
@@ -0,0 +1,88 @@
+package io.kestra.plugin.jdbc.rockset;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.jdbc.AbstractJdbcQuery;
+import io.kestra.plugin.jdbc.AbstractJdbcTrigger;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Wait for query on a Rockset database."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Wait for a sql query to return results and iterate through rows",
+            full = true,
+            code = {
+                "id: jdbc-trigger",
+                "namespace: io.kestra.tests",
+                "",
+                "tasks:",
+                "  - id: each",
+                "    type: io.kestra.core.tasks.flows.EachSequential",
+                "    tasks:",
+                "      - id: return",
+                "        type: io.kestra.core.tasks.debugs.Return",
+                "        format: \"{{json taskrun.value}}\"",
+                "    value: \"{{ trigger.rows }}\"",
+                "",
+                "triggers:",
+                "  - id: watch",
+                "    type: io.kestra.plugin.jdbc.rockset.Trigger",
+                "    interval: \"PT5M\"",
+                "    sql: \"SELECT * FROM my_table\""
+            }
+        )
+    }
+)
+public class Trigger extends AbstractJdbcTrigger {
+
+    @PluginProperty(dynamic = true)
```

Comment:
Make a interface to have a common documenation

Reply:
done